app/app.go

```diff
@@ -414,6 +415,27 @@ func New(
 	)
 	app.SetEndBlocker(app.EndBlocker)
 
+	planName := "v2.0.0"
```

Comment:
can this fetched from other place?
because two places hard-coded

Reply:
I think the plan name only specified once here.